src/main/java/com/iota/iri/service/spentaddresses/SpentAddressesProvider.java

```diff
@@ -8,6 +8,15 @@
  * Find, mark and store spent addresses
  */
 public interface SpentAddressesProvider {
+    /**
+     * Folder where we store spent address data
+     */
+    String SPENT_ADDRESSES_DB = "spent-addresses-db";
```

Comment:
see #1268

Reply:
yeah, I don't want to make it configurable for this release

I want to raise #1268 in a team meeting and decide on the best way to approach it.
Worst case we will add it on a later version